docs/api.md

```diff
@@ -1,5 +1,26 @@
 # API Reference
 
+This documents the API that serves out post graduate teacher training course,
+subject and training provider information. The version of the API documented
+here feeds the UCAS Apply system.
+
+# Table of Contents
+
+* [Authorisation](#authorisation)
+* [Retrieving Records](#retrieving-records)
+  * [Populating an empty system](#populating-an-empty-system)
```

Comment:
Indentation looks broken...

Reply:
I believe it's intentional:

<img width="593" alt="Screenshot 2019-03-18 at 10 29 31" src="https://user-images.githubusercontent.com/1650875/54524022-be524580-4968-11e9-80d5-611d7c4060f5.png">